### PR TITLE
Introducing our new pricing

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

**Review by Pierrci — APPROVED** [review]

File: pricing-update.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	 <a href="/pierric">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1624630689857-5e67de201009063689407481.jpeg?w=200&h=200&f=face" width="100" title="Gravatar">
+        <div class="bfc">
+            <code>pierric</code>
+            <span class="fullname">Pierric Cistac</span>
+        </div>
+    </a>
+</div>
+
+As you might have noticed, our [pricing page](https://huggingface.co/pricing) has changed a lot recently. In this post we will explain why we made those changes and what they mean for you.
```
Comment (Pierrci) [inline pricing-update.md:32]:
```suggestion
As you might have noticed, our [pricing page](https://huggingface.co/pricing) has changed a lot recently.
```
(maybe)

File: pricing-update.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Along with Inference Endpoints, we've recently introduced hardware upgrades for [Spaces](https://huggingface.co/spaces/launch). No subscription is required to use these services; you only need to add a credit card to your account from your [billing settings](https://huggingface.co/settings/billing). You can also attach a payment method to any of [your organizations](https://huggingface.co/settings/organizations).
+
+Your billing settings centralize everything about our paid services. From there, you can manage your personal PRO subscription, update your payment method, and visualize your usage for the past three months. Usage for all our paid services and subscriptions will be charged at the start of each month, and a consolidated invoice will be available for your records.
+
+**TL;DR**: **At HF we monetize by providing simple access to compute for AI**, with services like AutoTrain, Spaces and Inference Endpoints, directly accessible from the Hub. [Read more doc](https://huggingface.co/docs/hub/billing) about our pricing and billing system.
```
Comment (pcuenca) [inline pricing-update.md:40]:
```suggestion
**TL;DR**: **At HF we monetize by providing simple access to compute for AI**, with services like AutoTrain, Spaces and Inference Endpoints, directly accessible from the Hub. [Read more](https://huggingface.co/docs/hub/billing) about our pricing and billing system.
```

File: pricing-update.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+If you have any questions, feel free to reach out. We welcome your feedback 🔥
```
Comment (pcuenca) [inline pricing-update.md:42]: Maybe add a contact link? Not sure if email is the best option though.

```suggestion
If you have any questions, feel free to [reach out](mailto:team@huggingface.co). We welcome your feedback 🔥
```

**Review by pcuenca — APPROVED** [review]
Clear and concise, I like it!

File: pricing-update.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <span class="fullname">Pierric Cistac</span>
+        </div>
+    </a>
+</div>
+
+As you might have noticed, our [pricing page](https://huggingface.co/pricing) has changed a lot recently. In this post we will explain why we made those changes and what they mean for you.
+
+First of all, we are sunsetting the Paid tier of the Inference API service. The Inference API will still be available for everyone to use for free. But if you're looking for a fast, enterprise-grade inference as a service, we recommend checking out our brand new solution for this: [Inference Endpoints](https://huggingface.co/inference-endpoints).
+
+Along with Inference Endpoints, we've recently introduced hardware upgrades for [Spaces](https://huggingface.co/spaces/launch). No subscription is required to use these services; you only need to add a credit card to your account from your [billing settings](https://huggingface.co/settings/billing). You can also attach a payment method to any of [your organizations](https://huggingface.co/settings/organizations).
```
Comment (osanseviero) [inline pricing-update.md:36]:
```suggestion
Along with Inference Endpoints, we've recently introduced hardware upgrades for [Spaces](https://huggingface.co/spaces/launch), which allows running ML demos with the hardware of your choice. No subscription is required to use these services; you only need to add a credit card to your account from your [billing settings](https://huggingface.co/settings/billing). You can also attach a payment method to any of [your organizations](https://huggingface.co/settings/organizations).
```

**Review by osanseviero — APPROVED** [review]
Very nice!

File: pricing-update.md (hunk 3 above)
Comment (julien-c) [reply pricing-update.md:42]: team@ is maybe a bit spammy
